watchtower-plugin/src/main.rs

```diff
@@ -241,6 +291,29 @@ async fn retry_tower(
     }
 }
 
+/// Forgets about a tower wiping out all local data associated to it.
+async fn abandon_tower(
+    plugin: Plugin<Arc<Mutex<WTClient>>>,
+    v: serde_json::Value,
+) -> Result<serde_json::Value, Error> {
+    let tower_id = TowerId::try_from(v).map_err(|e| anyhow!(e))?;
+    let mut state = plugin.state().lock().unwrap();
+    if let Some(tower) = state.towers.get(&tower_id).clone() {
+        let result = if tower.status == TowerStatus::TemporaryUnreachable {
+            Err(anyhow!(
```

Comment:
I had the same thought while fixing this. I don't think it really matters given this will only end up printing on terminal, but I do agree it makes sense to return an `Ok(T)`. I'll fix this.

Related to the next two comments though, if we are going to change the log from `Abandoning...` to `Skipping already abandoned...`, then here we should always return `<tower_id> successfully abandoned`. Otherwise the `... Flagging it to be abandoned` won't make sense.

Reply:
Aha I meant we always return `<tower_id> successfully abandoned` in either way.

Reply:
> I had the same thought while fixing this. I don't think it really matters given this will only end up printing on terminal, but I do agree it makes sense to return an Ok(T). I'll fix it.

![image](https://user-images.githubusercontent.com/56920956/184869041-12b872ea-c2df-412a-bbc5-99a6e71cbd6f.png)
When I see this red x after executing a command, It alerts me that I did something wrong. I think many terminals too have something similar that visualizes the exit code of some command.

Reply:
Ohh, TIL!

Reply:
This is zsh with agnoster theme in `oh-my-zsh` 😉 .
I wouldn't recommend it though cuz it's kinda bloated.